candidate_models/base_models/__init__.py

```diff
@@ -499,4 +495,24 @@ def __init__(self):
             self[identifier] = LazyLoad(function)
 
 
+class StochasticModelPool(UniqueKeyDict):
+    """
+    Provides a set of stochastic models.
+    Each entry maps from `name` to an activations extractor.
+    """
+
+    def __init__(self):
+        super(StochasticModelPool, self).__init__(reload=True)
+
+        _key_functions = {
+            'voneresnet-50': lambda: voneresnet(model_name='resnet50'),
+            'voneresnet-50-robust': lambda: voneresnet(model_name='resnet50_at'),
```

Comment:
I think the CORnet-S version is missing from here (I see the changes further down, but no model)

Reply:
That's because the CORnet-S version is defined only at the brain-model level and is part of the VOneCORnetBrainPool in candidate_models/model_commitments/vonecornets

Reply:
ah got it